eyeshade/migrations/0006_payout_reports_ads/up.sql

```diff
@@ -0,0 +1,11 @@
+select execute($$
+
+create extension if not exists "uuid-ossp";
```

Comment:
This allows postgres to generate uuids for us instead of doing it in the app.

Reply:
i like it